### DISCUSSION TITLE
Split `USA` into `USA East` & `USA West` in the `World Hopper` plugin

### DISCUSSION THREAD

**Discussion by Felanbird** [runelite/runelite#16833] [Ideas]
Semi-recently Jagex implemented the flags to differentiate the East and West coast worlds, to which I _assume_ they also internally changed something for this change.

It would be very useful if this information could either be exposed, or utilized via the World Hopper plugin to allow the user to filter worlds based on this.

Alternatively a `W` or `E` could be drawn on the flag, as suggested in #13756, but honestly that's kind of an ugly approach.

**Comment by geheur** [comment]
Maybe it could show more stars if it was west coast, and more stripes if it was east coast. Not a lot of space to work with, so jagex's "put blue on the left side for west coast" thing is probably going to obscure the flag too much.

**Comment by brads93** [comment]
For me, the ping is enough of a tell for ppl who care to join either East or West coast US servers. I guess midwest is at the mercy of their geographical knowledge.

Also, I think the icons are currently too tiny and would need a rework to replicate the Jagex east/west icons which I didn't even understand until reading your post. I wouldn't mind, but it seems like it would be a tad more complex than the plugin needs to be.
![image](https://github.com/runelite/runelite/assets/1676751/4263baa3-43e4-455a-8754-41f6f09466f3)
![image](https://github.com/runelite/runelite/assets/1676751/039a88b6-088d-49bb-bda3-4d76363185f1)
